<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LC_LinePropertiesEditingWidget</class>
 <widget class="QWidget" name="LC_LinePropertiesEditingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="buttonGroup8">
     <property name="minimumSize">
      <size>
       <width>260</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Geometry</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="rightMargin">
       <number>6</number>
      </property>
      <item row="2" column="2" rowspan="2">
       <widget class="LC_PointPickButton" name="wPickPointEnd" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>23</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>23</width>
          <height>46</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Pick endpoint from the drawing</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lStartY">
        <property name="text">
         <string>Start point (y):</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <spacer name="spacer61">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leStartY">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Starting point Y coordinate</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lEndY">
        <property name="text">
         <string>End point (y):</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="leStartX">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Starting point X coordinate</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lStartX">
        <property name="text">
         <string>Start point (x):</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="leEndY">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Ending point Y coordinate</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="leEndX">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Ending point X coordinate</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="2">
       <widget class="LC_PointPickButton" name="wPickPointStart" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>23</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>23</width>
          <height>46</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Pick start point from the drawing</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <spacer name="spacer58">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lEndX">
        <property name="text">
         <string>End point (x):</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LC_PointPickButton</class>
   <extends>QWidget</extends>
   <header>lc_pointpickbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
